client/src/auth-wrapper.js

```diff
@@ -34,7 +34,9 @@ export const Auth0Provider = ({
 
       if (isAuthenticated) {
         const user = await auth0FromHook.getUser();
-        // const token = await auth0FromHook.getTokenSilently();
+        const token = await auth0FromHook.getTokenSilently();
+
+        localStorage.setItem('token', token);
```

Comment:
This could benefit from additional comments added in future code, to clarify the purpose to future build-ons.